tests/command.rs

```diff
@@ -39,8 +39,8 @@ fn parse_command_with_non_string_arg() {
 #[test]
 #[cfg(feature = "macros")]
 fn attribute_prefix() {
-    #[command(rename = "lowercase")]
     #[derive(BotCommand, Debug, PartialEq)]
+    #[command(rename = "lowercase")]
```

Comment:
There are a couple of warning of the same type next to this. Can you fix them too? (e.g.: line 59)

Reply:
Fixed.